util/dataset.go

```diff
@@ -32,13 +32,14 @@ var validTypes = map[string]string{
 }
 
 type DataSet struct {
-	ID             string   `config:"id" json:"id,omitempty" yaml:"id,omitempty"`
-	Title          string   `config:"title" json:"title" validate:"required"`
-	Release        string   `config:"release" json:"release"`
-	Type           string   `config:"type" json:"type" validate:"required"`
-	IngestPipeline string   `config:"ingest_pipeline,omitempty" config:"ingest_pipeline" json:"ingest_pipeline,omitempty" yaml:"ingest_pipeline,omitempty"`
-	Streams        []Stream `config:"streams" json:"streams,omitempty" yaml:"streams,omitempty" `
-	Package        string   `json:"package,omitempty" yaml:"package,omitempty"`
+	ID             string         `config:"id" json:"id,omitempty" yaml:"id,omitempty"`
+	Title          string         `config:"title" json:"title" validate:"required"`
+	Release        string         `config:"release" json:"release"`
+	Type           string         `config:"type" json:"type" validate:"required"`
+	IngestPipeline string         `config:"ingest_pipeline,omitempty" config:"ingest_pipeline" json:"ingest_pipeline,omitempty" yaml:"ingest_pipeline,omitempty"`
```

Comment:
Now that we have an `Elasticsearch` field, should `IngestPipeline` be moved under it?

Reply:
Good point. I would say yes!

Reply:
Will this pose a problem in terms of BWC for existing manifests?

Reply:
Yes and no. After 7.9 we must stop doing these changes but until then we can still update old ones. The way I plan such changes are:

* Update registry and support both format. Read in and output.
* Update Kibana and packages / integrations scripts
* Remove legacy code

@ycombinator Any help getting this changes in is appreciated ;-)

Reply:
Maybe I'm missing something about the plan but shouldn't there be an `IngestPipeline` field in the `Elasticsearch` struct below?

Reply:
Oh I see, the plan is to introduce it in #564.